packages/data-service-generator/src/server/resource/service/create-service.ts

```diff
@@ -217,7 +216,10 @@ async function createServiceBaseModule({
           (moduleContainer && !moduleContainer?.enabled && action) ||
           (action && !action.enabled)
         ) {
-          removeClassMethodByName(classDeclaration, action.name);
+          this.logger.debug(
+            `Removing ${action.name} from ${entityName} - not implemented yet`
+          );
```

Comment:
Is this correct or did you mean `import { logger as applicationLogger } from "./logging";` ?